src/Agent.Worker/Build/BuildDirectoryManager.cs

```diff
@@ -122,7 +122,25 @@ public sealed class BuildDirectoryManager : AgentService, IBuildDirectoryManager
             // Set the default clone path for each repository (the Checkout task may override this later)
             foreach (var repository in repositories)
             {
-                var repoPath = GetDefaultRepositoryPath(executionContext, repository, newConfig.SourcesDirectory);
```

Comment:
Is GetDefaultRepositoryPath method not actual anymore? (looks like no any usage of it in this file)

Reply:
Yeah, thanks! Removed it.